### DISCUSSION TITLE
[ERROR]: Failed to get disk_usage of drive None.

### DISCUSSION THREAD

**Discussion by geshenggang4** [ericaltendorf/plotman#621] [Q&A]
[config - 副本.yaml.txt](https://github.com/ericaltendorf/plotman/files/6526164/config.-.yaml.txt)
[debug.log](https://github.com/ericaltendorf/plotman/files/6526165/debug.log)

**Comment by geshenggang4** [comment]
HELP

**Reply by leonidcan** [reply]
It does not work on Windows. Needs to be Mac or Linux and Co
